### DISCUSSION TITLE
Welcome to sp2021_IDD_Interactive-Lab-Hub Discussions!

### DISCUSSION THREAD

**Discussion by hgimonet** [hgimonet/sp2021_IDD_Interactive-Lab-Hub#2] [General]
It's live!

**Comment by bnbeck** [comment]
Another product that would be super fun and useful to have.  Could you someone place the light within the faucet such that the water itself changes colors.  That would be useful for not only adults, but for promoting children to build the habits of healthy hand washing.  My concern with this is that if its battery operated it will require complicated disassembly of the faucet to replace, or if it is hard wired the concern with electricity mixing with water.  Furthermore, general corrosion and wear could see this device have a short lifecycle before needing to be replaced.

**Comment by IreneFP** [comment]
Very timely!!! Super cool. I like the idea of a small light as an add on. Maybe the water itself can change color when you are done. Maybe, thinking about adding the sound and to gamify it a little bit, you can think about incorporating this: https://www.scripps.org/news_items/6922-coronavirus-prevention-your-20-second-handwashing-playlist
